skimage/segmentation/_felzenszwalb_cy.pyx

```diff
@@ -125,3 +125,118 @@ def _felzenszwalb_grey(image, double scale=1, sigma=0.8,
         flat = flat[flat]
     flat = np.unique(flat, return_inverse=True)[1]
     return flat.reshape((height, width))
+
+
+def _felzenszwalb_rgb(image, double scale=1, sigma=0.8,
+                       Py_ssize_t min_size=20):
+    """Felzenszwalb's efficient graph based segmentation for multiple channels.
+
+    Produces an oversegmentation of a multi-channel image using a fast, minimum spanning
+    tree based clustering on the image grid.
+    The number of produced segments as well as their size can only be
+    controlled indirectly through ``scale``. Segment size within an image can
+    vary greatly depending on local contrast.
+
+    Parameters
+    ----------
+    image: ndarray
```

Comment:
add a space between the variable and the column -> image : ndarray. + Elsewhere.